transform/openshift/cmd.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+
+	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/konveyor/crane-lib/transform"
+	"github.com/konveyor/crane-lib/transform/cli"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func main() {
```

Comment:
I am wondering if this should be in a new dir say examples/cmd/openshift???

Reply:
yes totally. don't think this might belong in this repo. I have it as https://github.com/konveyor/crane-lib/pull/10/commits/5f5328db0918bbf9c902516b58f3dccabd7add06 do not merge code, just to see if this makes sense